elasticache/redis/Acornfile

```diff
@@ -0,0 +1,88 @@
+name:        "AWS Elasticache Redis"
+description: "Amazon's Redis compatible Elasticache cluster"
+
+args: {
+	// Name assigned to the cluster during creation. Default value is "RedisCluster".
+	clusterName: "RedisCluster"
+
+	// Key value pairs to apply to all resources.
+	tags: {}
+
```

Comment:
What is the snapshot behavior here? Or any backups?

Reply:
Not implemented yet. AWS does support [backup to S3](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/backups.html).